packages/orchestrator/lib/clients/client.integration.test.ts

```diff
@@ -62,6 +62,34 @@ describe('OrchestratorClient', async () => {
             });
             expect(res.isOk()).toBe(true);
         });
+        it('should be updatable', async () => {
+            const name = nanoid();
+            await client.recurring({
+                name,
+                state: 'STARTED',
+                startsAt: new Date(),
+                frequencyMs: 300_000,
+                groupKey: nanoid(),
+                retry: { count: 0, max: 0 },
+                timeoutSettingsInSecs: { createdToStarted: 30, startedToCompleted: 30, heartbeat: 60 },
+                args: {
+                    type: 'sync',
+                    syncId: 'sync-a',
+                    syncName: nanoid(),
+                    syncJobId: 5678,
+                    connection: {
+                        id: 123,
+                        connection_id: 'C',
+                        provider_config_key: 'P',
+                        environment_id: 456
+                    },
+                    debug: false
+                }
+            });
+
+            const res = await client.updateSyncFrequency({ scheduleName: name, frequencyMs: 600_000 });
```

Comment:
would be great to check that it's been actually updated

Reply:
true but there is an unit test in scheduler that is testing the internal function is correctly updating. This test is more focus on confirming that the communication between the client and the scheduler service is successful, assuming that what' internal to the scheduler is working as expected. Soon I am gonna need to add a get schedule functionality to the client. I will update this test to check the value has been updated